iOS/SampleApp/SampleApp/Services/TextViewsInitializer.swift

```diff
@@ -0,0 +1,78 @@
+// Copyright 2019 Flying Vehicle Monster team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+
+extension ViewController : UITextViewDelegate {
+    fileprivate static let TERMINATING_CHAR = "\n"
+    fileprivate static let DESCRIPTION_ID = "Description"
+    fileprivate static let DESCRIPTION_VALUE = "FVM description"
+    fileprivate static let DAMAGED_PARTS_ID = "DamagedCarParts"
+    fileprivate static let DAMAGED_PARTS_VALUE = "Hood; Roof;"
```

Comment:
Is that some type of example?

Reply:
Default values in SampleApp. So when you run the app and change nothing, framework gets "FVM Description" and these two parts (Hood & Roof) are highlighted